services/compute/compute.go

```diff
@@ -217,9 +217,26 @@ type DynamicMemoryConfiguration struct {
 	TargetMemoryBuffer *uint32 `json:"targetmemorybuffer,omitempty"`
 }
 
+type Assignment string
+
+// possible values of gpu assignment
+const (
```

Comment:
Why are we having to redefine the constants here? are they not available in the moc repo?

Reply:
Redefine is used to translate int32 in moc to string. Supposing it will make sdk more user friendly and avoid exposing underlying moc details to customers. It's following the "SecurityTypes" definition (also translate int32 in moc to string in moc-sdk-for-go).